frontend/app/src/next/hooks/use-members.tsx

```diff
@@ -87,8 +96,35 @@ export function MembersProvider({ children }: { children: React.ReactNode }) {
       }
     },
     onSuccess: async () => {
-      await membersQuery.refetch();
-      await invitesQuery.refetch();
+      await queryClient.invalidateQueries({ queryKey: ['tenant-members:list', tenantId] });
```

Comment:
same here re: `next` - I was wondering how this worked and then I noticed the hook it was using